### ISSUE TITLE
the text "*/* */*" glitch lists and verbatim

### ISSUE THREAD

**Issue by pudinha** [ruby/rdoc#455]
this will correctly create a verbatim, a definition list with a bullet list and a verbatim with text starting with *:

```
  */*

---

asd ::
  * asd
  * asd
  * asd

---

  * asd
  * asd
  * asd
```

however, if i add another `*/*` somewere inside a verbatim, on teh same line, another line, or another verbatim, or even in a new paragraph, everything gets messed up. teh bullet list will become inline text and all teh starting asterisks of teh last verbatim will disapear.

it seems that github doesnt suffer from this, just `rdoc` locally.

this is wat i did:

```
$ rdoc --version
5.1.0
$ rdoc
Parsing sources...
100% [ 1/ 1]  a.rdoc

Generating Darkfish format into /home/rogi/a/doc...

  Files:      1

  Classes:    0 (0 undocumented)
  Modules:    0 (0 undocumented)
  Constants:  0 (0 undocumented)
  Attributes: 0 (0 undocumented)
  Methods:    0 (0 undocumented)

  Total:      0 (0 undocumented)
    0.00% documented

  Elapsed: 0.0s

```

**Comment by duckinator** [comment]
I believe I've encountered the same problem, or a related one, and came up with a testcase.

Assuming a `test.rb` that contains this:

```ruby
# x
#   a/**/*a
#   b/**/*b
#   c/**/*c
#   d/**/*d
#   e/**/*e
def foo; end
```

Running `rdoc test.rb` and opening `doc/Object.html` reveals this:

![Screenshot_20200625_165134](https://user-images.githubusercontent.com/39698/85794182-2e22b480-b704-11ea-915d-d772924393dd.png)

Adding more similar lines seems to work, but the first 3 are always garbled.

**Comment by aycabta** [comment]
@pudinha @duckinator Thank you for reporting. This is fixed in 6.2.0 or later.
